projects/swimlane/ngx-ui/src/lib/components/json-editor/json-editor-flat/json-editor-flat.component.ts

```diff
@@ -77,18 +77,22 @@ export class JsonEditorFlatComponent extends JsonEditor implements OnInit, OnCha
   }
 
   onPropertyConfig(): void {
-    this.dialogService.create({
+    const dialog = this.dialogService.create({
       template: this.propertyConfigTmpl,
       context: {
         property: this.schema,
         schema: this.schema,
-        formats: this.customFormats
+        formats: this.customFormats,
+        apply: (options: PropertyConfigOptions) => {
```

Comment:
Nice fix. I wonder why the dialog stopped closing.

Reply:
Before the component itself was calling `destroyAll` which was not good when this component is used in a dialog.